### PR TITLE
Fi 54 join command channel sending messages

### PR DESCRIPTION
**Pull request by ynslee**
I will keep working on this branch. So channel name that has # in the middle of it or so hasn't been properly tested yet. I will test and push it with other changes later on. Now it would work if you just do join hello or join #hello

### CODE REVIEW

**Review by jestebanpelaez18 — APPROVED** [review]
Nicee

File: srcs/commands/Join.cpp [hunk 1]
```diff
@@ -104,6 +107,11 @@ static int clientErrorChecks(Client *client, std::map<std::string, Channel*> &ch
 {
 	std::string hostname = client->getHostName();
 
+	if (channelName.empty())
```
Comment (hussojo) [inline srcs/commands/Join.cpp:110]: Is this something that should go to buffer or straight to client?

Comment (ynslee) [reply srcs/commands/Join.cpp:110]: It's an error message that they don't have it or I haven't found it yet. so it should go straight to client

File: srcs/commands/Join.cpp [hunk 2]
```diff
@@ -6,10 +6,13 @@ static std::string getChannelName(std::string channel)
 	std::size_t found = channel.find('#');
```
Comment (hussojo) [inline srcs/commands/Join.cpp:6]: We also have a channel class getter that iss named the same, can we use it, or name this one differently?

**Review by hussojo — APPROVED** [review]

**Pull request merged**